src/sync/pull-request.ts

```diff
@@ -87,14 +87,8 @@ export const getPullRequestTask = async (
 			edgesWithCursor.map(async (pull) => {
 				const prResponse = await gitHubInstallationClient.getPullRequest(repository.owner.login, repository.name, pull.number);
 				const prDetails = prResponse?.data;
-				const ghUser = await getGithubUser(gitHubInstallationClient, prDetails?.user.login
-				);
-				const data = await transformPullRequest(
-					{ pullRequest: pull, repository },
-					prDetails,
-					gitHubInstallationClient.baseUrl,
-					ghUser
-				);
+				const	reviews = await getPullRequestReviews(gitHubInstallationClient, repository, pull, logger);
+				const data = await transformPullRequest(gitHubInstallationClient, prDetails, reviews, logger);
```

Comment:
Should we maybe feature-flag this change?

Reply:
Think I agree with Boris. Better to mitigate the chances of it going pear-shaped.

Reply:
Yeah thats not unreasonable, will slap in a FF, and have the old transform as false and this new one on true.
please hold.

Reply:
Flag added, updated tests to test true/false paths